plugins/modules/aci_tenant_action_rule_profile.py

```diff
@@ -249,18 +361,64 @@ def main():
             module_object=action_rule,
             target_filter={"name": action_rule},
         ),
+        child_classes=list(child_classes.keys()),
     )
 
     aci.get_existing()
 
     if state == "present":
+        child_configs = []
+        for key, value in child_classes.items():
+            if value[0] is not None:
+                if value[0] == "" or value[0] == False or check_all_none_values_dict(value[0]):
+                    if isinstance(aci.existing, list) and len(aci.existing) > 0:
+                        for child in aci.existing[0].get("rtctrlAttrP", {}).get("children", {}):
+                            if child.get(key):
+                                child_configs.append(
+                                    {
+                                        key:dict(
+                                            attributes=dict(status="deleted"),
+                                        ),
+                                    }
+                                )
+                elif value[0] != "" or value[0] == True or value[0] != {}:
```

Comment:
think this can be shortened

```suggestion
                elif value[0] == True or not value[0]:
```